src/pages/index.js

```diff
@@ -129,26 +249,65 @@ function getCardElement(data) {
   cardImageElement.src = data.link;
   cardImageElement.alt = data.name;
 
+  if (localStorage.getItem(`liked-${data._id}`) === "true") {
```

Comment:
Please, don't use `localStorage` for liking because in the other computer, you'll not see red hearts if you visit the site there


You should use `isLiked` for checking if it's liked or not


```js
  if (data.isLiked) {
    cardLikeBtn.classList.add("card__like-button_liked");
  }

```